src/main/java/teammates/ui/controller/InstructorFeedbackResultsPageData.java

```diff
@@ -222,26 +222,27 @@ public void initForSectionPanelViews(InstructorAttributes instructor,
         
         switch (viewType) {
             case RECIPIENT_GIVER_QUESTION:
-                Map<String, Map<String, List<FeedbackResponseAttributes>>> sortedResponsesForRGQ = 
-                        bundle.getResponsesSortedByRecipientGiverQuestion();
+                Map<String, Map<String, List<FeedbackResponseAttributes>>> sortedResponsesForRGQ =
+                    bundle.getResponsesSortedByRecipientGiverQuestion(isGroupedByTeam());
```

Comment:
Might be unrelated, but is it possible to make `isGroupedByTeam()` a field instead of a method?